druid/adhoc-jobs/pom.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.ekstep</groupId>
+    <artifactId>adhoc-jobs</artifactId>
+    <version>1.0</version>
+
+    <properties>
+        <spark.version>2.0.1</spark.version>
+        <scala.version>2.11.8</scala.version>
+        <scala.maj.version>2.11</scala.maj.version>
+        <scoverage.plugin.version>1.1.1</scoverage.plugin.version>
```

Comment:
Are we using the scoverage plugin to generate code coverage? If not, please remove the dependency.

Reply:
done